less/shared.less

```diff
@@ -164,6 +164,17 @@ input[type="submit"] {
   float: left;
   padding-left: 5px;
   padding-right: 5px;
+  word-wrap: break-word;
```

Comment:
Not related to RTL either (sorry should probably have done separate PRs)
This is to fix long words that are cut. Footer of https://donate.mozilla.org/de/one-page, should be “Datenschutzerklärung”:
<img width="147" alt="capture d ecran 2017-01-24 a 01 44 31" src="https://cloud.githubusercontent.com/assets/1294206/22230649/c4ecce8e-e1d6-11e6-952d-ca1e93f74834.png">